rmw_fastrtps_cpp/src/rmw_node.cpp

```diff
@@ -217,7 +228,19 @@ rmw_create_node(
   Domain::getDefaultParticipantAttributes(participantAttrs);
 
   participantAttrs.rtps.builtin.domainId = static_cast<uint32_t>(domain_id);
+  // since the participant name is not part of the DDS spec
   participantAttrs.rtps.setName(name);
+  // the node name is also set in the user_data
```

Comment:
I don't think so since the previous line sets the name on the participant and only the following lines set it in the user data.

Reply:
Sorry that was unclear, I meant the other way around:
- we set the name in participant name
- then we say "// since the participant name is not part of the DDS spec the node name is also set in the user_data"
- the we set it in user_data

Does that make sense?

Reply:
Currently the first comment makes clear that setting the participant name is not covered by the spec - right when it is setting that attribute.

Reply:
fair enough, let's leave it as is then